src/performance.jl

```diff
@@ -1,18 +1,19 @@
 using DataFrames
+using CSVFiles
 #this file contains code to run a speed test of multiple Monte Carlo runs against speeds of PAGE09 run in Excel
 #PAGE09 was run on an an Intel Core i7-4770 @3.4 GHz CPU with 16GB of main memory
 #care should be taken in comparing speeds from runs of Mimi-PAGE on different computers
 
 include("mimi-page/src/montecarlo.jl")
 
-df = DataFrame([String,Bool,Int,Float64],[:model,:multithreading,:runs,:elapsed],0)
+df = DataFrame([String,String,Int,Float64],[:model,:multithreading,:runs,:elapsed],0)
 
 for i in (1_000, 10_000)
     timing = do_monte_carlo_runs(i)
 
-    push!(df, ("Mimi-PAGE", false, i, timing))
+    push!(df, ("Mimi-PAGE", "false", i, timing))
```

Comment:
Is this for working with the CSV package?

Reply:
Yes, for some odd reason it doesn't detect ``Bool`` columns properly...

Reply:
I've run into a lot of problems with the CSV package.  I wish they had offered a package with the same functionality as was in DataFrames.

Reply:
Which one are you trying? There are two: CSV.jl and CSVFiles.jl. The latter is maintained by me and what we are using here. The ``Bool`` thing clearly needs to be fixed...

Reply:
Oh, that's why you're using the load and save functions.  Sorry, I was thinking of the CSV.jl package, which I've temporarily given up on.  I should try FileIO.